lib/Page.js

```diff
@@ -193,29 +194,58 @@ Page.prototype.prepareTemplateData = function () {
   };
 };
 
+/**
+ * Gets the closest header to an element.
+ * @param $ html content
+ * @param headingsSelector jQuery selector for selecting headings
+ * @param element to find closest heading
+ */
+function getClosestHeader($, headingsSelector, element) {
```

Comment:
`getClosestHeading` and fix all wrongly used `headers`.

Don't confuse Headers and Headings: http://www.altformat.org/index.asp?pid=389

Reply:
TIL, thanks!